plugins/meeting/src/index.ts

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import type { Readable } from 'svelte/store'
+
+import { getPlugin, plugin } from '@anticrm/platform'
+import type { Asset } from '@anticrm/status'
+import type { Plugin, Service } from '@anticrm/platform'
+import type { Space } from '@anticrm/core'
+import { Peer as RawPeer } from '@anticrm/webrtc'
+
+export interface RoomSpace extends Space {}
+
+export interface Peer extends RawPeer {
+  peer: RTCPeerConnection
+  media: MediaStream
+  isMediaReady: boolean
+}
+
+export type Status = 'joining' | 'joined' | 'left'
+
+export interface Room {
+  peers: Readable<Peer[]>
+  user: Readable<Peer>
+  screen: Readable<Peer>
+  status: Readable<Status>
+}
+
+export interface MeetingService extends Service {
+  room: Room
+  join: (room: string) => Promise<void>
+  leave: () => Promise<void>
+  shareScreen: () => Promise<void>
+  stopSharingScreen: () => Promise<void>
```

Comment:
`Service` may become obsolete in future versions, do you really need this methods available to other plugins?

Reply:
Yeah, good point, made all this stuff internal until we get some real cases